apps/televisit-demo/backend/endmeeting/index.js

```diff
@@ -0,0 +1,22 @@
+const AWS = require('aws-sdk');
+const chime = new AWS.Chime({ region: 'us-east-1' });
+
+exports.handler = async event => {
+  const meetingId = event.queryStringParameters.meetingId;
+  console.info('Deleting meeting');
+  await chime
+  .deleteMeeting({
+    MeetingId: meetingId
+  })
+  .promise();
+  return {
+    statusCode: 200,
+    headers: {
+      'Access-Control-Allow-Headers': 'Authorization',
+      'Access-Control-Allow-Origin': '*',
```

Comment:
non-blocking:
- add env variable
- hook it up with cloudfront url 

if not- add call out in readme